packages/lib-classifier/src/components/Classifier/components/TaskArea/TaskArea.js

```diff
@@ -41,19 +41,24 @@ class TaskArea extends React.Component {
     }
     return (
       <Tabs
-        onActive={this.onTabClick.bind(this)}
+        background='white'
```

Comment:
We shouldn't set the background color to just white. It varies on the light/dark theme.

Reply:
Yup, but light / dark isn't in the TESS milestone.

Reply:
Incidentally, there are a few different methods for altering style based on theme being used, so we should settle on one in the post-TESS retrospective.